src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -117,7 +117,7 @@ private Recipe getRecipe(List<Recipe> lastShownList, Index recipeIndex) throws C
             return lastShownList.get(zeroBasedIndex);
         }
 
-        throw new CommandException(Messages.MESSAGE_INVALID_RECIPE_DISPLAYED_INDEX);
+        throw new CommandException(Messages.MESSAGE_INVALID_RECIPE_INDEX);
```

Comment:
I'm assuming this change was only to the variable name to make it clearer?

Reply:
yep!